docs/source/zero-code-change.rst

```diff
@@ -0,0 +1,209 @@
+may propagate differently and lead to different outcomes. This can be
+exacerbated by discretization operations in which values end up in
+different categories based on floating point values.
+
+Secondarily, some algorithms are implemented in a fundamentally different
+way on GPU than on CPU in order to make efficient use of the GPU's highly
+parallel compute capabilities. In such cases, ``cuml.accel`` will translate
+hyperparameters appropriately to maintain equivalence with the CPU
+implementation. Differences of this kind are noted in the corresponding entry
+of `Zero Code Change Limitations <0cc_limitations.rst>`_ for that
+estimator.
+
+If you discover a use case where the quality of results obtained with
+``cuml.accel`` is worse than that obtained without, please `report it as a bug
+<https://github.com/rapidsai/cuml/issues/new?template=bug_report.md>`_, and the
+RAPIDS team will investigate.
+
+5. How much faster is ``cuml.accel``?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This depends on the individual algorithm being accelerated and the dataset
+being processed. As with cuML itself, you will generally see the most benefit
+when ``cuml.accel`` is used on large datasets. Please see
+`Zero Code Change Benchmarks <0cc_benchmarks.rst>`_ for some representative benchmarks.
+
+Please note that the first time an estimator method is called in a Python
+process, there may be some overhead due to JIT compilation of cupy kernels. To
+get an accurate sense of performance, run the method once on a small subset of
+data before measuring runtime on a full-scale dataset.
+
+6. Will I run out of GPU memory if I use ``cuml.accel``?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+``cuml.accel`` will use CUDA `managed memory <https://developer.nvidia.com/blog/unified-memory-cuda-beginners/>`_ for allocations on NVIDIA GPUs. This means that host memory can be used to augment GPU memory, and data will be migrated automatically as necessary. This does not mean that ``cuml.accel`` is entirely impervious to OOM errors, however. Very large datasets can exhaust the entirety of both host and device memory. Additionally, if device memory is heavily oversubscribed, it can lead to slow execution. ``cuml.accel`` is designed to minimize both possibilities, but if you observe OOM errors or slow execution on data that should fit in combined host plus device memory for your system, please `report it <https://github.com/rapidsai/cuml/issues/new?template=bug_report.md>`_, and the RAPIDS team will investigate.
+
+7. What is the relationship between ``cuml.accel`` and ``cudf.pandas``?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Both projects serve a similar role. Just as ``cuml.accel`` offers zero code
+change acceleration for Scikit-Learn and similar packages, ``cudf.pandas``
+offers zero code change acceleration for Pandas. They can be used together by
+TODO(wphicks): FILL THIS IN ONCE THIS MECHANISM HAS BEEN IMPLEMENTED.
```

Comment:
Reminder to remove the TODO from the doc. Could also be nice to add a link to `cudf.pandas`.